src/integrationTest/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionControllerRequestTranscriptionIntTest.java

```diff
@@ -240,6 +245,112 @@ void transcriptionRequestWithDuplicateValuesWithNoTimes() throws Exception {
+        String expectedJson = """
+            {
+              "type": "TRANSCRIPTION_110",
+              "title": "Transcription could not be requested, no audio",
+              "status": 404
+            }
+            """;
+        JSONAssert.assertEquals(expectedJson, actualJson, JSONCompareMode.NON_EXTENSIBLE);
+    }
+
+    @Test
+    @Order(15)
+    void transcriptionRequestStartTimeOutsideHearing() throws Exception {
+        TranscriptionUrgencyEnum transcriptionUrgencyEnum = TranscriptionUrgencyEnum.STANDARD;
+        TranscriptionTypeEnum transcriptionTypeEnum = TranscriptionTypeEnum.SENTENCING_REMARKS;
+
+        TranscriptionRequestDetails transcriptionRequestDetails = createTranscriptionRequestDetails(
+            hearing.getId(), courtCase.getId(), transcriptionUrgencyEnum.getId(),
+            transcriptionTypeEnum.getId(), TEST_COMMENT, now().minusHours(1), now().plusMinutes(10)
+        );
+
+        dartsDatabaseStub.save(hearing);
+
+        MockHttpServletRequestBuilder requestBuilder = post(ENDPOINT_URI)
+            .header("Content-Type", "application/json")
+            .content(objectMapper.writeValueAsString(transcriptionRequestDetails));
+
+        MvcResult mvcResult = mockMvc.perform(requestBuilder)
+            .andExpect(status().isNotFound())
+            .andReturn();
+
+        String actualJson = mvcResult.getResponse().getContentAsString();
+        assertFailedTranscription111Error(actualJson);
+
+        assertAudit(0);
+    }
+
+    @Test
+    @Order(16)
+    void transcriptionRequestEndTimeOutsideHearing() throws Exception {
+        TranscriptionUrgencyEnum transcriptionUrgencyEnum = TranscriptionUrgencyEnum.STANDARD;
+        TranscriptionTypeEnum transcriptionTypeEnum = TranscriptionTypeEnum.SENTENCING_REMARKS;
+
+        TranscriptionRequestDetails transcriptionRequestDetails = createTranscriptionRequestDetails(
+            hearing.getId(), courtCase.getId(), transcriptionUrgencyEnum.getId(),
+            transcriptionTypeEnum.getId(), TEST_COMMENT, now().plusMinutes(1), now().plusHours(10)
+        );
+
+        dartsDatabaseStub.save(hearing);
+
+        MockHttpServletRequestBuilder requestBuilder = post(ENDPOINT_URI)
+            .header("Content-Type", "application/json")
+            .content(objectMapper.writeValueAsString(transcriptionRequestDetails));
+
+        MvcResult mvcResult = mockMvc.perform(requestBuilder)
+            .andExpect(status().isNotFound())
+            .andReturn();
+
+        String actualJson = mvcResult.getResponse().getContentAsString();
+        assertFailedTranscription111Error(actualJson);
+
+        assertAudit(0);
+    }
+
+    private void assertFailedTranscription111Error(String actualJson) {
+        String expectedJson = """
+            {
+              "type": "TRANSCRIPTION_111",
+              "title": "Transcription could not be requested, times outside of hearing times",
+              "status": 404
```

Comment:
I know it says in the ticket, but I'm not sure if these should be 404s? I would probably expect 400? CC @jackmaloney @hemantasharma1129

Reply:
Yeh happy with 400 if you think it's more appropriate.